### PR TITLE
chore: add contribution guidelines document

### PR DESCRIPTION
**Pull request by schalkneethling**
This adds the beginnings of our contribution guidelines. We will build upon this and might end up making these guidelines organisation wide over time.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+> The act or process of selecting and bringing together people or groups for a specific purpose
+
+Now if you replace people with content, that would be an accurate description of what curation means in the context of the web://workbook.
+
+With that said, curation does not mean copying. While you can, and should, use quotations from works that explain a topic well, there are a couple of points to remember:
+
+1. Never copy large portions of text verbatim i.e. using exactly the same words; corresponding word for word.
+2. Always reference, and link to your sources.
+3. Almost put quoted text as quotes in the document.
```
Comment (bpires) [inline CONTRIBUTING.md:19]: Didn't get the meaning of "Almost" here @schalkneethling. Would you mind elaborating a bit? It seems to be a matter of English/idiom 😅 Also, wouldn't quoted text already come as quotes?

**Review by bpires — COMMENTED** [review]
Very important to set this guidelines.

**Comment by bpires** [comment]
@schalkneethling, I added code of conduct and license references. Please take a look.

**Review by bpires — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (schalkneethling) [reply CONTRIBUTING.md:19]: Oh, that is a typo :)

```suggestion
3. Always put the quoted text as a block quote. [Block quotes in Markdown](https://commonmark.org/help/tutorial/05-blockquotes.html)
```

**Pull request merged**